### PR TITLE
Updated the usercases for invalid balance or no accounts and a way to…

### PR DESCRIPTION
**Pull request by sidstedt**
## Description

Made it so if user enters 'avsluta' user will return to menu. Checks if user have any accounts to transfer from and balance on account.  Checks if user to transfer to have any accounts.

## Related Issue ( optional )

Link to any related issue

## Changes

Describe the changes in more detail, if necessary include explanation of why some choices were made.

## Screenshots & GIFs ( optional )

media to provide more visual context if necessary

## Notes & Comments

General notes and comments, such as if there is things to consider ahead of deployment

### CODE REVIEW

File: RebelAllianceBank/Users/Customer.cs [hunk 1]
```diff
@@ -113,46 +113,72 @@ public void CreateAccount()
...
-                Console.WriteLine("Skriv namnet på det konto du vill föra över från:");
+                Console.WriteLine("Skriv namnet på det konto du vill föra över från, skriv 'avsluta' för att återgå till menyn");
                 foreach (var account in _bankAccounts)
                 {
                     Console.WriteLine($"{account.AccountName} (Saldo: {account.Balance:N2})");
                 }
                 string currentUserAccountName = Console.ReadLine();
                 currentUserAccount = _bankAccounts.FirstOrDefault(account => account.AccountName.Equals(currentUserAccountName, StringComparison.OrdinalIgnoreCase));
-                if (currentUserAccount == null)
+                if ("avsluta" == currentUserAccountName.ToLower())
```
Comment (tantting) [inline RebelAllianceBank/Users/Customer.cs:139]: You still need to catch null somehow in your if-statements. Right now, null. crashes the program

**Review by tantting — COMMENTED** [review]
I need to press enter twice when writing avsluta. Seem to be an additional Ready() etc somewhere

File: RebelAllianceBank/Users/Customer.cs (hunk 1 above)
Comment (sidstedt) [reply RebelAllianceBank/Users/Customer.cs]: Fixed! now it should work as intended

**Comment by sidstedt** [comment]
> I need to press enter twice when writing avsluta. Seem to be an additional Ready() etc somewhere
And if you step it you will se it aint my fault, someone planted a ReadKey() in the CustomerMenu section 😂

**Review by tantting — APPROVED** [review]
Well done!

**Pull request merged**